File: bin/basic_training/Day12_2.js
function solution(num_list) {
	return num_list.findIndex((value) => value < 0);
}

// 자바스크립트 배열.findIndex((value, index) => 조건식)
// 조건식을 만족하는 배열의 첫번째 요소에 대한 인덱스를 반환
// 만족하는 요소가 없으면 -1 을 반환 

File: src/basic_training/Day12_2.java
package basic_training;

public class Day12_2 {
	public int solution(int[] num_list) {
		int count = 0;
		for (int num : num_list) {
			if (num < 0) {
				return count;
			}
			count++;
		}
		return -1;
	}
}

// 번외 문제풀이 _ IntStream
// return IntStream.range(0, num_list.length).filter(i -> num_list[i] < 0).findFirst().orElse(-1);